api/types/constants.go

```diff
@@ -345,6 +345,15 @@ const (
 	// OriginKubernetes is an origin value indicating that the resource was
 	// created from the Kubernetes Operator.
 	OriginKubernetes = "kubernetes"
+
+	// AWSAccountIDLabel is used to identify nodes by AWS account ID
+	// found via automatic discovery, to avoid re-running installation
+	// commands on the node.
+	AWSAccountIDLabel = TeleportNamespace + "/account-id"
+	// AWSInstanceIDLabel is used to identify nodes by EC2 instance ID
+	// found via automatic discovery, to avoid re-running installation
+	// commands on the node.
+	AWSInstanceIDLabel = TeleportNamespace + "/instance-id"
```

Comment:
I've been thinking, when we add auto-enrollment on other clouds (Azure, GCP), how do we differentiate which cloud the node originated from? These labels are pretty generic. How about we add another label, e.g. `teleport.dev/cloud` and set it to `aws` for auto-discovered EC2 instances?

Reply:
The script isn't really cloud-dependent though, not sure we need different scripts just to pass an extra label.

It may be ok as-is though, this issue can probably be solved separately by adding support for cloud nodes to report their own metadata in heartbeats, then we can get rid of setting these labels in the script explicitly also. Let's keep it for now.